tarantool_test.go

```diff
@@ -3550,6 +3550,72 @@ func TestConnection_NewWatcher(t *testing.T) {
 	}
 }
 
+func newWatcherReconnectionPrepareTestConnection(t *testing.T) (*Connection, context.CancelFunc) {
+	t.Helper()
+
+	const server = "127.0.0.1:3015"
+	testDialer := dialer
+	testDialer.Address = server
+
+	inst, err := test_helpers.StartTarantool(test_helpers.StartOpts{
+		Dialer:       testDialer,
+		InitScript:   "config.lua",
+		Listen:       server,
+		WaitStart:    100 * time.Millisecond,
+		ConnectRetry: 10,
+		RetryTimeout: 500 * time.Millisecond,
+	})
+	t.Cleanup(func() { test_helpers.StopTarantoolWithCleanup(inst) })
+	if err != nil {
+		t.Fatalf("Unable to start Tarantool: %s", err)
+	}
+
+	ctx, cancel := test_helpers.GetConnectContext()
+
+	reconnectOpts := opts
+	reconnectOpts.Reconnect = 100 * time.Millisecond
+	reconnectOpts.MaxReconnects = 0
+	reconnectOpts.Notify = make(chan ConnEvent)
+	conn, err := Connect(ctx, testDialer, reconnectOpts)
+	if err != nil {
+		t.Fatalf("Connection was not established: %v", err)
+	}
+
+	test_helpers.StopTarantool(inst)
+
+	// wait for reconnection process to be started
```

Comment:
Comments should start with a capital letter and end with a dot.
```suggestion
	// Wait for reconnection process to be started.
```